lib/codecs/src/encoding/format/json.rs

```diff
@@ -35,6 +35,11 @@ impl JsonSerializer {
     pub const fn new() -> Self {
         Self
     }
+
+    /// Encode event as JSON.
+    pub fn encode_json(&self, event: Event) -> Result<serde_json::Value, serde_json::Error> {
+        event.try_into()
```

Comment:
This doesn't seem like it's going to match the structure of the `encode` method below, but will include the `Event` wrapper that's used in `NativeJsonSerializer`.

Having these kinds of similar-but-different methods bring the danger of these types of bug, so it'd be nice to at least see some tests that the outputs are equivalent.

Reply:
Oh damn, that's a great catch!

Fixed and added tests in 0cba4ba9c78fed9e0d0735eed008771dc2ee8a42

Reply:
Oh damn, that's a great catch!

Fixed and added tests in 0cba4ba9c78fed9e0d0735eed008771dc2ee8a42